### ISSUE TITLE
Line endings prevent launching Packager_GNU.sh

### ISSUE THREAD

**Issue by DianaLunaXIV** [EternalllZM/Packager#2]
After following the directions to download the packager files, when trying to run the script, I had this error:
`bin/bash^M: No such file or directory`

Some research pointed to this being an issue of line endings (Windows line endings are CRLF vs Unix line endings are LF). Simply copying and pasting the script as provided with Unix line endings enabled in VSCode resolved this.

**Comment by EternalllZM** [comment]
I was afraid of this, I was attempting to upload through my VM last night and it would simply not allow it. I will reupload and repackage all of the aforementioned contents. Did the script run fine afterward?

**Comment by EternalllZM** [comment]
I have just pushed a fix to the actual script itself, and reuploaded the Packager contents for the GNU variant. Can you please test and confirm that it is working now? Thanks

**Comment by DianaLunaXIV** [comment]
To answer all of your questions:

- I was able to run the script perfectly fine after I changed the endings.
- I cloned the repo and tested the script you just pushed and it works just fine!
- I also tested the release (fully redownloaded, extracted, and then followed Linux steps on your readme) and it properly detected an update. Since it's essentially redownloading the script itself, I followed the directions to exit after the script pulled the update, re-enabled executing permissions, and ran it again. Once again, the script worked as intended!
